README.md

```diff
@@ -116,6 +116,16 @@ Everything you need is available prebuild in FreeBSD' ports.
 
     pkg install gmake node
 
+#### Arch:
+
+You can install cjdns by running
+
+    pacman -S cjdns
+
+If you need to build from source, everything you need can be installed like this
+
+    pacman -S nodejs git base-devel
```

Comment:
Why would we advise that people use the unofficial one instead of the one
already in the repos?

2016-08-08 11:04 GMT-07:00 Łukasz Magiera <notifications@github.com>:

> In README.md
> <https://github.com/cjdelisle/cjdns/pull/941#discussion_r73923440>:
>
> > @@ -116,6 +116,16 @@ Everything you need is available prebuild in FreeBSD' ports.
> >
> >      pkg install gmake node
> >
> > +#### Arch:
> > +
> > +You can install cjdns by running
> > +
> > +    pacman -S cjdns
> > +
> > +If you need to build from source, everything you need can be installed like this
> > +
> > +    pacman -S nodejs git base-devel
>
> Why not point to AUR package https://aur.archlinux.org/packages/cjdns-git/
> via pacaur like pacaur -S cjdns-git?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cjdelisle/cjdns/pull/941/files/9d36b63af1f2973bea06c8826776237f425f9aa7#r73923440>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABQw1FGT436CnuzWyIzvABDcPG9QH1Frks5qd2-fgaJpZM4I-cyi>
> .
>